opensrp-reveal/src/test/java/org/smartregister/reveal/shadow/SyncStatusBroadcastReceiverShadow.java

```diff
@@ -0,0 +1,21 @@
+package org.smartregister.reveal.shadow;
+
+import org.mockito.Mockito;
+import org.robolectric.annotation.Implementation;
+import org.robolectric.annotation.Implements;
+import org.smartregister.receiver.SyncStatusBroadcastReceiver;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Implements(SyncStatusBroadcastReceiver.class)
+public class SyncStatusBroadcastReceiverShadow {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'SyncStatusBroadcastReceiverShadow' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=5414138)